modules/nf-core/ctree/main.nf

```diff
@@ -0,0 +1,56 @@
+process CTREE {
+    tag "$meta.id"
+    label "process_low"
```

Comment:
Just to check, how much computational power does this tool require? low will default to 2 cpus and 12GB RAM, is that sufficient?